.github/workflows/ci.yml

```diff
@@ -252,13 +283,72 @@ jobs:
           name: nuget-packages
           path: ./Bin/Release/ILGPU*.${{ steps.package.outputs.version }}.*nupkg
 
-    outputs:
-      version: ${{ steps.version.outputs.version }}
+  # Create the Samples zip file, replacing ILGPU project references with equivalent NuGet package references.
+  package-samples:
+    needs: [check-version]
```

Comment:
can be simplified: `needs: check-version`

Reply:
there is one or two more as well

Also these
```
needs: [a,b,c]
```
Can become
```
needs:
  - a
  - b
  - c
```
optional formatting improvements ⚡

Reply:
Done.